test/MorseL.Shared.Tests/ServicesMocker.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using MorseL.Extensions;
+using MorseL.Scaleout;
+using MorseL.Sockets;
+
+namespace MorseL.Shared.Tests
+{
+    public class ServicesMocker
+    {
+        public readonly Mock<IServiceProvider> ServiceProviderMock = new Mock<IServiceProvider>();
+        public readonly Mock<IServiceScopeFactory> ServiceScopeFactoryMock = new Mock<IServiceScopeFactory>();
+        public readonly Mock<IBackplane> BackplaneMock = new Mock<IBackplane>();
+        public readonly Mock<IOptions<MorseLOptions>> MorseLOptionsMock = new Mock<IOptions<MorseLOptions>>();
+        public readonly Mock<ILoggerFactory> LoggerFactoryMock = new Mock<ILoggerFactory>();
+        public readonly Mock<IChannel> ChannelMock = new Mock<IChannel>();
+
+        private readonly Mock<IServiceProvider> HubActivatorMockProvider = new Mock<IServiceProvider>();
+
+        public ServicesMocker()
+        {
+            ServiceProviderMock.Setup(m => m.GetService(It.Is<Type>(t => t == typeof(IServiceScopeFactory))))
```

Comment:
You should be able to just do:
```suggestion
            ServiceProviderMock.Setup(m => m.GetService(typeof(IServiceScopeFactory)))
```

Reply:
Oh really? Nice! I'll fix that up while I'm working on getting all tests unskipped.